src/message/Message.ts

```diff
@@ -0,0 +1,72 @@
+import type { Payloads, payloadCodeUnion } from "@utils/index";
+import { isObject } from "@utils/index";
```

Comment:
It could, but `@typescript-eslint/consistent-type-imports` makes sure that type imports are separated from imports that are used at runtime. (that's not documented, ive opened an issue in typescript-eslint's monorepo to document that)